sdks/python/src/opik/api_objects/trace.py

```diff
@@ -203,6 +204,13 @@ def log_feedback_score(
 class TraceData:
     """
     The TraceData object is returned when calling :func:`opik.opik_context.get_current_trace_data` from a tracked function.
+
```

Comment:
@alexkuzmik These get documented in the [public docs](https://www.comet.com/docs/opik/python-sdk-reference/Objects/TraceData.html), let's move these to a comment in the code itself